jetty9-base/pom.xml

```diff
@@ -112,7 +136,7 @@
               <goal>copy-dependencies</goal>
             </goals>
             <configuration>
-              <outputDirectory>${project.build.directory}/jetty-base/lib/gcp</outputDirectory>
+              <outputDirectory>${project.build.directory}/jetty-base/lib/gcloud</outputDirectory>
```

Comment:
@meltsufin no I don't believe so.  The lib/gcloud dir is the location that  standard jetty places the gcloud jars for session management so this is the same in flex.